CHANGES

```diff
@@ -1,130 +1,144 @@
-Changelog
-=========
+# Changelog
 
-Version 0.1:                                                      March 3, 2014
--------------------------------------------------------------------------------
-- Initial release
+## Version 1.0.0                                               February 10, 2023
```

Comment:
You already did the hard work but the right-justified date may be more trouble than its worth in the markdown version (i.e., we could just do `Version 1.0.0 (February 10, 2023)` or something)

Reply:
Yeah, I realized after I did it that it's a zero for effort.  I'll update it

Reply:
updated